brave/src/main/java/brave/propagation/ExtraFieldPropagation.java

```diff
@@ -52,11 +52,12 @@
  *   ExtraFieldPropagation.newFactoryBuilder(B3Propagation.FACTORY)
  *                        .addField("x-vcap-request-id")
  *                        .addPrefixedFields("baggage-", Arrays.asList("country-code", "user-id"))
+ *                        .build()
  * );
  *
  * // Later, you can call below to affect the country code of the current trace context
- * ExtraFieldPropagation.current("country-code", "FO");
```

Comment:
@marcingrzejszczak based on your feedback, I think simple get/set is better right?